tests/ops/fixtures/saas/shopify_fixtures.py

```diff
@@ -100,3 +104,153 @@ def shopify_dataset_config(
     )
     yield dataset
     dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def shopify_erasure_data(
+    shopify_connection_config, shopify_erasure_identity_email, shopify_secrets
+) -> Generator:
+    """
+    Creates a dynamic test data record for erasure tests.
+    Yields customer ID as this may be useful to have in test scenarios
```

Comment:
nit: i'd probably update this comment since it's more than just customer ID that's yielded

Reply:
Nice catch, I'll update it too in new commit.